prov/gni/include/gnix.h

```diff
@@ -74,6 +74,10 @@ extern "C" {
 #include "gnix_mr_cache.h"
 #include "gnix_mr_notifier.h"
 
+#if HAVE_UDREG
+#include "udreg_pub.h"
```

Comment:
More pickiness:  I think we should use brackets, as below, instead of double quotes.